ops/lib.rs

```diff
@@ -524,6 +558,16 @@ fn is_fast_typed_array(arg: impl ToTokens) -> bool {
   RE.is_match(&tokens(arg))
 }
 
+fn is_fast_v8_value(
+  core: &TokenStream2,
+  arg: impl ToTokens,
+) -> Option<TokenStream2> {
+  match tokens(&arg).contains("serde_v8 :: Value") {
+    false => None,
+    true => Some(quote! { #core::v8::fast_api::Type::V8Value }),
+  }
```

Comment:
Nit: use if statement instead of match

Reply:
Or just use [`bool::then`](https://doc.rust-lang.org/std/primitive.bool.html#method.then)